tools/schemacode/schemacode/validator.py

```diff
@@ -228,13 +228,19 @@ def load_top_level(
 
 def load_entities(
     my_schema,
+    inheritance_regex=r".*?\\\.json.*?",
```

Comment:
https://bids-specification.readthedocs.io/en/latest/02-common-principles.html#the-inheritance-principle talks also abut .bvec and .tsv but those aren't to be implemented in this PR I would assume.  Let's then add a TODO comment

```suggestion
    # TODO: support for .tsv and .bvec files when validator starts to care about their content
    inheritance_regex=r".*?\\\.json.*?",
```

or because we are not here to check for content etc, may be right away add support for those .bvec and .tsv? (I guess should be taken from schema regexes for where those files already used).  But then I am not sure if it is worth adding it as an option to the function -- do you see it getting some other values?

Reply:
but also why there is `.*?` trailer in regex -- could it be `.json.gz` or alike?

Reply:
fixed formats, and addressed `.*?` ambiguity in [9d0b4e3](https://github.com/bids-standard/bids-specification/pull/1120/commits/9d0b4e336149f039fa9435ff1a4e414cfa60aca4).